### DISCUSSION TITLE
Updating nuxt from 3.0.0 to 3.7.x

### DISCUSSION THREAD

**Discussion by abelozerceva** [nuxt/nuxt#23292] [Questions] [answered]
Good day. There was a desire in the project to update nuxt to the latest current stable version. Currently using version 3.0.0-rc.8, which I consider to be quite old. 
Question: is it still worth upgrading the version and if so, how can this be done with minimal risk of breaking the project?

**Comment by TheAlexLichter** [comment, marked as answer]
Yes! Upgrading from any kind of RC is highly advisable (actually, the latest minor version is always advisable given bug fixes and features).

**Comment by sauravbv** [comment]
I did the same migration from `3.0.0` to `3.7.1` and it went pretty well. Not too many breaking changes
@abelozerceva
